src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+function getAverageAge(sex) {
```

Comment:
Consider renaming the 'sex' parameter to 'peopleOfSpecificSex' to make the code more descriptive and avoid confusion.